app/page.tsx

```diff
@@ -29,20 +30,29 @@ export default function Home() {
   }, [isSubmitted]);
 
   return (
-    <div className="lg:ml-20">
-      <span className="hidden"><LinktaLogoWithText /></span>
-      <KeyValueProposition/>
-      <div className='flex flex-col sm:flex-row sm:justify-between lg:ml-10'>
-        <Suspense fallback={<Loading />}>
-          <div className="sm:w-2/3 sm:order-2 z-0">
-            <LandingPageTreeVisualizationPanel />
-          </div>
-        </Suspense>
-        <section className="w-full flex flex-col justify-between min-w-[360px] sm:w-1/3 sm:order-1 mt-2 mb-12">
-          <article className='max-w-[430px] z-10'>
+    <div className="contained-div flex flex-col">
+      {/* <span className="hidden"><LinktaLogoWithText /></span> */}
+      <HeadlineProposition/>
+      <div className='linkta-main-content flex flex-col align-items-center md:flex-row px-10 md:justify-around'>
+        {/* <Loading /> */}
+        <section className="linkta-form text-xl sm:text-lg min-w-9 md:basis-1/2">
+          <article className=''>
             {isSubmitted ? <EmailVerificationPrompt /> : <PrelaunchSignUpForm handleSuccessfulSubmit={handleSuccessfulSubmit} />}
           </article>
         </section>
+        {/* <Suspense fallback={<Loading />}> */}
+          <div className="linkta-image flex self-center md:basis-1/2 mb-8 md:mb-0">
```

Comment:
I think the static image looks great.  Just a quick thought: What having the image stack on top of the form rather than below on smaller screens?

Reply:
@colinvandergraaf Honestly, that crossed my mind a few times but I just kept it as it was before. I presume the idea was that the call to action was more important than the cute 3D model.